lib/nexpose/node.rb

```diff
@@ -18,7 +18,7 @@ def initialize(xml_node)
     def supported_tags
       [
         # attributes
-        :address, :status, :device_id, :hardware_address,
+        :address, :status, :device_id, :hardware_address, :site_name,
```

Comment:
This wasn't the case with the original list, but can we have this in alpha order so it's easier to parse mentally (e.g. "is :device_id in the list already or not?")

Reply:
Also, shouldn't :site_name be now part of the "fields" for this plugin?

Reply:
@etdsoft resolved, thank you!